analysis/generate_report.py

```diff
@@ -167,7 +170,29 @@ def generate_report(experiment_names,
         experiment_df = data_utils.clobber_experiments_data(
             experiment_df, experiment_names)
 
+    # Generate coverge reports for each benchmark.
+    benchmark_names = experiment_df.benchmark.unique()
     fuzzer_names = experiment_df.fuzzer.unique()
+    coverage_report_directory = os.path.join(report_directory,
+                                             'coverage-reports')
+    if not in_progress:
```

Comment:
I don't see why we should use `in_progress` for this. Sure we don't actually need it while it's in progress but this feels like exposing details of the dispatcher to report generation that shouldn't be exposed. There should probably be a seperate variable.